boards/README.md

```diff
@@ -4,36 +4,79 @@ Platforms Supported by Tock
 
+Tock divides boards into three approximate 'tiers' of support.
+These tiers are newly defined and are a bit informal as a result,
+but the approximate definitions:
+
+ - **Tier 1:** The most feature-complete and thoroughly tested boards.
+               Loosely, these are boards used most regularly by core
+               team members or other highly engaged contributors.
+ - **Tier 2:** Platforms seeing reasonably regular use. These generally
+               have broader, but still incomplete, peripheral support.
+               They may also be 'relatives' of Tier 1 boards (e.g. a
+               less-used varient in the nrf52 family) – likely in good
+               shape, but not heavily tested. Some Tier 2 boards may
+               have known issues, which are documented in release notes
+               during release testing.
+ - **Tier 3:** New or highly experimental. These should support the
+               minimum platform requirements laid out in
+               [the Porting documentation](../doc/Porting.md), but
+               make no promises beyond that.
+
+ - **Tier Z:** These are unstable _platforms_, where Tock bringup is
+               happening in parallel with hardware development. These
+               are highly experimental by their nature.
+
+### Tier 1
+
 | Board                                                             | Architecture     | MCU            | Interface  | App deployment              | QEMU Support? |
 |-------------------------------------------------------------------|------------------|----------------|------------|-----------------------------|---------------|
 | [Hail](hail/README.md)                                            | ARM Cortex-M4    | SAM4LC8BA      | Bootloader | tockloader                  | No            |
 | [Imix](imix/README.md)                                            | ARM Cortex-M4    | SAM4LC8CA      | Bootloader | tockloader                  | No            |
-| [Nordic nRF52-DK](nordic/nrf52dk/README.md)                       | ARM Cortex-M4    | nRF52832       | jLink      | tockloader                  | No            |
 | [Nordic nRF52840-DK](nordic/nrf52840dk/README.md)                 | ARM Cortex-M4    | nRF52840       | jLink      | tockloader                  | No            |
 | [Nordic nRF52840-Dongle](nordic/nrf52840_dongle/README.md)        | ARM Cortex-M4    | nRF52840       | jLink      | tockloader                  | No            |
+
+### Tier 2
+
+| Board                                                             | Architecture     | MCU            | Interface  | App deployment              | QEMU Support? |
+|-------------------------------------------------------------------|------------------|----------------|------------|-----------------------------|---------------|
+| [Nordic nRF52-DK](nordic/nrf52dk/README.md)                       | ARM Cortex-M4    | nRF52832       | jLink      | tockloader                  | No            |
```

Comment:
I think the `nRF52DK` might easily be Tier 1.

Reply:
Does anyone regularly use this variant? I have a few, but I have abandoned them in favor of nrf52840dk's. If you're using plain 52's, I'm happy to promote it, but otherwise I think it should stay Tier 2 for the same reason the dongle is here – nrf family, very likely supported, but not continuously used / tested.

Reply:
I was using that almost exclusively over the `nRF52840DK`, but I did not bring this board with me during my move, so starting virtually now I'll switch to the `nRF52840DK` as well I suppose. If no one else really uses that, that's good enough reasoning for me to keep it in Tier 2.